CMakeLists.txt

```diff
@@ -1,3 +1,17 @@
+# This is the main CMake file for fv3atm.
+#
+# Mark Potts, Kyle Gerheiser, Alex Richert
```

Comment:
Is this author list correct? I think Dusan is the author.

Reply:
Fixed.